ethereum/api/build.gradle

```diff
@@ -125,7 +125,7 @@ dependencies {
 
 artifacts { testSupportArtifacts testSupportJar }
 
-task generateTestBlockchain() {
+tasks.register('generateTestBlockchain') {
```

Comment:
not a gradle expert, is this just a code style change or has it also other impacts?

Reply:
IntelliJ keeps tagging it as a lint issue.